packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -106,6 +106,8 @@ Resources:
               - kms:ScheduleKeyDeletion
               - kms:CancelKeyDeletion
               - kms:GenerateDataKey
+              - kms:TagResource
+              - kms:UntagResource
```

Comment:
Hello,

From the issue I can understand why the [`kms:TagResource`](https://docs.aws.amazon.com/cli/latest/reference/kms/tag-resource.html) is required.

>the update fails if tags are added.

`"Access denied for operation 'TagResource'."`


Mind providing the use case that requires untagging/updating?

Cheers!

Reply:
#21571 goes into detail about the use case where a tag changes.

Though I admit I made an assumption that the uderlying Cloudformation would attempt an `UntagResource` action at some point during the updating process.